.github/workflows/ci-test-coverage.yml

```diff
@@ -0,0 +1,28 @@
+name: ci-test-coverage
+
+on:
+  workflow_run:
+    workflows: [ci-test-local, ci-test-ginkgo]
+    types:
+      - completed
+
+jobs:
+    codecoverage:
+      name: Measure code coverage
+      runs-on: ubuntu-20.04
+      steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.workflow_run.head_branch }}
+      - run: git branch
+      - run: env  
+      
+      - name: Download artifacts
+        uses: actions/download-artifact@v3
+        with: 
+          name: coverage
+      - name: Upload to CodeCov
+        uses: codecov/codecov-action@v3 
+        with:
+          files: ./KubeArmor/gover.coverprofile
```

Comment:
I think this won't be generate for now, so it's fine if it's empty. Let's come back to this later and might as well remove the dependency for it.